src/tests/Forums.test.js

```diff
@@ -0,0 +1,11 @@
+import {render} from '@testing-library/react';
+import Forums from '../componentsTest/Forums';
+import React from "react"; 
+
+describe('Forums list', () => {
+    it('loads forums on first render', () => {
```

Comment:
add 1-3 more tests -> a lot of them are reusable and require one/two line changes, look at the tests in https://github.com/Safewaan/HonkedIn/pull/32 for reference